src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -651,6 +661,30 @@ private void sendReportTrackToEditor(String eventName) {
             }
         }
 
+        private void sendLayoutError(JSONObject errorMessage) {
+            if (mEditorConnection == null || !mEditorConnection.isValid()) {
+                return;
+            }
+
+            final OutputStream out = mEditorConnection.getBufferedOutputStream();
+            final OutputStreamWriter writer = new OutputStreamWriter(out);
+
+            try {
+                errorMessage.put("type", "layout_error");
```

Comment:
We also have an "envelope" protocol for this sort of thing (With "type" and "payload" as top level keys) - in fact, we already have an error message format, it might be good to use that format if possible.